components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1709,6 +1765,24 @@ public void deleteGroup(String groupId) throws NotFoundException, CharonExceptio
         }
     }
 
+    /**
+     * Method to interpret startIndex as 1 when the startIndex equals to NULL in the request.
+     *
+     * @param startIndex StartIndex in the request.
+     * @return Updated startIndex
+     */
+    private int handleStartIndexEqualsNULL(Integer startIndex) {
+
+        if (startIndex == null) {
+            if (log.isDebugEnabled()) {
+                log.debug("NULL argument for startIndex in the request. NULL value interpreted as 1 and updated "
```

Comment:
Shall we summarize this debug log to reflect the operation, to something like `NULL value returned for the startIndex argument interpreted as 1`?

Reply:
fixed: 2a9df39b802d0a818f4c26bdd0f6c7ffe48a4136